README.md

```diff
@@ -28,6 +28,16 @@ Run `make` to build `./bin/cloudwatch.so`. Then use with Fluent Bit:
 * `role_arn`: ARN of an IAM role to assume (for cross account access).
 * `auto_create_group`: Automatically create the log group. Valid values are "true" or "false" (case insensitive). Defaults to false.
 * `endpoint`: Specify a custom endpoint for the CloudWatch Logs API.
+* `credentials_endpoint`: Specify a custom HTTP endpoint to pull credentials from. The HTTP response body should look like the following:
+```
+{
+    "AccessKeyId": "ACCESS_KEY_ID",
+    "Expiration": "EXPIRATION_DATE",
+    "RoleArn": "TASK_ROLE_ARN",
```

Comment:
Lookslike SDK does not support RoleArn in endpointcreds. Shall we remove it?

https://docs.aws.amazon.com/sdk-for-go/api/aws/credentials/endpointcreds/